cypress/support/fragments/invoices/invoices.js

```diff
@@ -94,11 +94,26 @@ export default {
     InteractorsTools.checkCalloutMessage(invoiceStates.invoiceLineCreatedMessage);
   },
 
-  checkInvoiceLine: (invoiceLine) => {
+  createInvoiceLineFromPol: (orderNumber, rowNumber = 0) => {
+    const checkBoxLabel = `record ${rowNumber} checkbox`;
+    cy.do([
+      Accordion({ id: invoiceLinesAccordionId }).find(actionsButton).click(),
+      Button('Add line from POL').click()
+    ]);
+    cy.expect(Modal('Select order lines').exists());
+    cy.do([
+      Modal('Select order lines').find(SearchField({ id: 'input-record-search' })).fillIn(orderNumber),
+      Modal('Select order lines').find(searchButton).click(),
+      Checkbox({ ariaLabel: checkBoxLabel }).clickInput(),
```

Comment:
you click by row index. Text has requirement "Select POL from the Search results list". May be it's related, that you need to click by row with special POL(nod just by special line index)

Reply:
I'm filtering all POLs so result list contains only one record - that's why it's safe to select by index.

Reply:
will it still stable in case of parallel tests execution(autotests/manua checks in bugfest)?

Reply:
Yes, because I'm filtering by unique order created in this test. That's why we have unique combination of order-order line- invoice and it can't be affected by other tests